### ISSUE TITLE
Got an error after upgrading form 2.24.0 to 2.29.0

### ISSUE THREAD

**Issue by lauthu** [evermeer/EVReflection#81]
I got an error after upgrading form 2.24.0 to 2.29.0, it seems that after upgrade, EVReflection begins to parse property name and add underscore. For example, I have below class:

```
 class A : EVObject{
        var openId : String
}
```

and an instance of class `A` will be serialized as:

```
{
     'open_id' : value
}
```

now... So I downgrade to 2.24.0, it worked again.

**Comment by evermeer** [comment]
Yesterday evening I pushed version 2.30.0 to GitHub and CocoaPods that should solve this issue.

With the release of 2.29.0 I implemented conversionOptions that replaced the keyCleanup parameter. On a couple of locations where keyCleanup was meant to be false i implemented the keyCleanup as a conversionOption by mistake. This should now be solved

Please let me know if this issue is fixed with the 2.30.0 release

**Comment by lauthu** [comment]
@evermeer I tried 2.30.0, and the problem still exist.

**Comment by evermeer** [comment]
☺️ That was a (de)serialize mixup. I just pushed a fix with update 2.30.1

The following test will now succeed:

```
    func testIssue81() {
        let a = A81()
        a.openId = "value"
        let json = a.toJsonString()
        print(json)
        XCTAssertEqual(json, "{\n  \"openId\" : \"value\"\n}", "Incorrect serialisation to json")
    }
}


class A81: EVObject {
    var openId: String = ""
}
```

**Comment by lauthu** [comment]
@evermeer 2.30.1 works well. 😊 Thank you.
